test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -59,7 +59,9 @@ public class bug8033699 {
 
     public static void main(String[] args) throws Throwable {
         robot = new Robot();
-
+        SwingUtilities.invokeAndWait(() -> {
+            focusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager();
+        });
         // Get all installed Look and Feels
         UIManager.LookAndFeelInfo[] lafs = UIManager.getInstalledLookAndFeels();
```

Comment:
```suggestion
            focusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager();
        });

        UIManager.LookAndFeelInfo[] lafs = UIManager.getInstalledLookAndFeels();
```
I'd add a blank here, too, and remove the comment since `getInstalledLookAndFeels` is already self-documenting.